specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-02-01-preview/elastic.json

```diff
@@ -65,6 +65,52 @@
         }
       }
     },
+    "/providers/Microsoft.Elastic/listElasticVersions": {
```

Comment:
>listElasticVersions [](http://example.com/codeflow?start=34&length=19)

should be a noun not a verb

Reply:
/providers/Microsoft.Elastic/elasticVersions

Reply:
For this API path, "https://management.azure.com/providers/Microsoft.Elastic/elasticVersions", would I need to do anything other than manifest rollout for it to function.

Reply:
I dont follow the connection between my comment nd your question